lms/static/js/views/message_banner.js

```diff
@@ -0,0 +1,32 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone'
+    ], function (gettext, $, _, Backbone) {
+
+        var MessageBannerView = Backbone.View.extend({
+
+            initialize: function (options) {
+                this.template = _.template($('#message_banner-tpl').text());
+            },
+
+            render: function () {
+                this.$el.html(this.template({
+                    message: this.message
+                }));
+                return this;
+            },
+
+            showMessage: function (message) {
+                this.message = message;
+                this.render();
+            },
+
+            hideMessage: function () {
+                this.$el.html('');
```

Comment:
I think it would make sense to do in this function:
```
this.message = nil;
this.render();
```

and in render do:
```
if (this.message === null) {
  this.$el.html('');
} else {
// the standard render
}
```